application/helpers/expressions/em_manager_helper.php

```diff
@@ -8548,8 +8550,17 @@ static function ProcessCurrentResponses()
                                     }
                                 }
                                 break;
-#                            case 'N': //NUMERICAL QUESTION TYPE
-#                            case 'K': //MULTIPLE NUMERICAL QUESTION
+                            case 'N': //NUMERICAL QUESTION TYPE
+                            case 'K': //MULTIPLE NUMERICAL QUESTION
+                                if (trim($value)!="") {
+                                    $aAttributes=$LEM->getQuestionAttributesForEM($LEM->sid, $qid, $_SESSION['LEMlang']);
+                                    if (!isset($aAttributes[$qid])) {
+                                        $aAttributes[$qid]=array();
+                                    }
+                                    if($aAttributes[$qid]['thousands_separator']=='1') {
+                                        $value=str_replace($invertRadix['separator'], '', $value);
+                                    }
+                                }
```

Comment:
This part is to show "thousand seperator" without JS, right ? But thousand don't really work without JS. Really needed ?
-> js : update value after shown (.ready)
-> no js : get the value, and show the real value

I do a quick test with separator dot : user can not enter 1,000 without JS. Then : no js : no thousand_separator.